app/src/main/res/layout/info_card_fragment.xml

```diff
@@ -1,203 +1,186 @@
-
-                <View
-                    android:id="@+id/divider"
-                    android:layout_width="match_parent"
-                    android:layout_height="1dp"
-                    android:layout_marginBottom="15dp"
-                    android:background="#DEBCC0"
-                    android:backgroundTint="#DEBCC0"
-                    tools:visibility="visible" />
-
-                <LinearLayout
-                    android:id="@+id/departments"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:orientation="vertical">
-
-                    <TextView
-                        android:id="@+id/departments_title"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:gravity="center"
-                        android:text="Departments"
-                        android:textColor="#932339"
-                        android:textColorHighlight="#FFFFFF"
-                        android:textColorHint="#FFFFFF"
-                        android:textColorLink="#FFFFFF"
-                        android:textSize="16sp" />
-
-                    <TextView
-                        android:id="@+id/departments_list"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="10dp"
-                        android:text="TextView" />
-                </LinearLayout>
-
-                <LinearLayout
-                    android:id="@+id/services"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:orientation="vertical">
-
-                    <TextView
-                        android:id="@+id/services_title"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="10dp"
-                        android:gravity="center"
-                        android:text="Services"
-                        android:textColor="#932339"
-                        android:textSize="16sp" />
-
-                    <TextView
-                        android:id="@+id/services_list"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:layout_marginTop="10dp"
-                        android:layout_marginBottom="20dp"
-                        android:text="TextView" />
-                </LinearLayout>
-
-            </LinearLayout>
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:gravity="center"
+                android:text="Services"
+                android:textColor="#932339"
```

Comment:
i think you should add these hex in a style inside color.xml since we might reuse them in the future

Reply:
you can even create styles with colors, text, and more all inside and just pass the style to a component, it helps with reusability

Reply:
this also applies to all the previous cases where you used a hex color directly

Reply:
ok will do, thanks!